app/components/CapitalProjectPanel/index.tsx

```diff
@@ -0,0 +1,116 @@
+export { CapitalProjectDetail } from "./CapitalProjectDetail";
+export { CapitalCommitmentsTimeline } from "./CapitalCommitmentsTimeline";
+export { CapitalCommitmentsTable } from "./CapitalCommitmentsTable";
+
+import {
+  Collapse,
+  Flex,
+  Heading,
+  IconButton,
+  Show,
+} from "@nycplanning/streetscape";
+import {
+  CapitalProjectDetail,
+  CapitalProjectDetailProps,
+} from "./CapitalProjectDetail";
+import { useState } from "react";
+import { MobilePanelResizeBar } from "../MobilePanelResizeBar";
+import {
+  CapitalCommitmentsTable,
+  CapitalCommitmentsTableProps,
+} from "./CapitalCommitmentsTable";
+import { ChevronDownIcon, ChevronUpIcon } from "@chakra-ui/icons";
+
+export interface CapitalProjectPanelProps
+  extends CapitalProjectDetailProps,
+    CapitalCommitmentsTableProps {}
+
+export function CapitalProjectPanel(props: CapitalProjectPanelProps) {
+  const [commitmentsAreVisible, setCommitmentsAreVisible] = useState(false);
+
+  const toggleCommitmentsAreVisible = () =>
+    setCommitmentsAreVisible((commitmentsAreVisible) => !commitmentsAreVisible);
+
+  return (
+    <Flex
+      borderTopRadius={"base"}
+      borderBottomRadius={{ base: "0", lg: "base" }}
+      background={"white"}
+      direction={"column"}
+      width={{ base: "full", lg: "21.25rem" }}
+      maxW={{ lg: "unset" }}
+      boxShadow={"0px 8px 4px 0px rgba(0, 0, 0, 0.08)"}
+      position={{ base: "fixed", lg: "static" }}
+    >
+      <MobilePanelResizeBar
+        isExpanded={commitmentsAreVisible}
+        isExpandedToggle={toggleCommitmentsAreVisible}
+        marginTop={"0.75rem"}
+      />
+      <CapitalProjectDetail
+        capitalProject={props.capitalProject}
+        agencies={props.agencies}
+        navigationBtn={props.navigationBtn}
+        capitalCommitments={props.capitalCommitments}
+        onNavigationClick={props.onNavigationClick}
+      />
+      <Show below="lg">
+        <Collapse in={commitmentsAreVisible}>
+          <Flex paddingY={3} direction={"column"} maxHeight={"30vh"}>
+            <Heading
+              fontSize="sm"
+              fontWeight={"bold"}
+              paddingX={3}
+              paddingBottom={3}
+            >
+              Commitment Details
+            </Heading>
+            <CapitalCommitmentsTable
+              capitalCommitments={props.capitalCommitments}
+              capitalCommitmentTypes={props.capitalCommitmentTypes}
+            />
+          </Flex>
+        </Collapse>
+      </Show>
+      <Show above="lg">
+        <Flex
+          direction={"column"}
+          backgroundColor="gray.50"
+          paddingX={4}
+          paddingBottom={4}
```

Comment:
As a stop-gap for the vertical centering issue we discussed, you can change this to `paddingBottom={commitmentsAreVisible ? 4 : 0}` and it seems to work as expected (or at least closer than the current version).

Reply:
updated